docker-compose.yaml

```diff
@@ -7,14 +7,16 @@ services:
         ports:
             - '3000:3000'
         ###### uncomment below as appropriate
-        # depends_on:
-        #     - mongo
+        depends_on:
+            - mongo
         #     - postgresql
         #     - rabbitmq
         #     - redis
     ###### uncomment the correponsing db's/services you need
-    # mongo:
-    #     image: mongo:4.2.8-bionic
+    mongo:
+        image: mongo:4.2.8-bionic
+        ports:
+            - 27017:27017
```

Comment:
refer to previous comment

Reply:
Basically you want docker.compose and docker.compose.test to run instances of mongoDB at different ports, correct?

Reply:
Yeah, basically.